### PR TITLE
Add ssh_node function and increase timeout

### PR DESCRIPTION
**Pull request by deepshikhaaa**
This commit includes:
- Add another function to check the SSH connections on the remote machines
- Increase the timeout of waiting for the server to restart

### CODE REVIEW

File: distributed-tests/rackspace-server-manager.py [hunk 1]
```diff
@@ -55,21 +54,21 @@ def create_node(nova, counts):
...
         else:
-            ips.append(ip_address)
+            ips.append(str(ip_address))
 
 
     for ip in reversed(ips):
-        ret = ping_node(ip)
-        timeout = time.time()
+        ret = ssh_node(ip)
+        timeout = time.time() + 300
```
Comment (nigelbabu) [inline distributed-tests/rackspace-server-manager.py]: This looks convoluted. How about this.

```
timeout = 0

while ret != 0 and timeout < 300:
    # run commands
    time.sleep(5)
    timeout = timeout + 5

File: distributed-tests/rackspace-server-manager.py [hunk 2]
```diff
@@ -55,24 +58,19 @@ def create_node(nova, counts):
             print 'No IP address assigned!'
             sys.exit(1)
         else:
-            ips.append(ip_address)
+            ips.update({str(ip_address):False})
```
Comment (nigelbabu) [inline distributed-tests/rackspace-server-manager.py]: can't you just do `ips[ip_address] = False` instead?

File: distributed-tests/rackspace-server-manager.py [hunk 3]
```diff
@@ -55,24 +58,19 @@ def create_node(nova, counts):
...
-
-    print 'The list of alive servers is: {0}'.format(ips)
+    timeout = 0
+    ret = check_ssh(ips)
+    while ret != True and timeout < 300:
+        ret = check_ssh(ips)
+        time.sleep(5)
+        timeout = timeout + 5
+    reachable_ip = [ip for ip, val in ips.items() if val==True]
+    print 'The list of reachable servers: {0}'.format(ip)
```
Comment (nigelbabu) [inline distributed-tests/rackspace-server-manager.py]: Might want to print the unreachable Ips instead.

Comment (deepshikhaaa) [reply distributed-tests/rackspace-server-manager.py]: May need to print reachable Ips also in the console logs so as to further debug easily.

**Pull request merged**